lib/src/upgrader.dart

```diff
@@ -519,7 +567,7 @@ class Upgrader {
           TextButton(
               child: Text(messages!.message(UpgraderMessage.buttonTitleLater)!),
               onPressed: () => onUserLater(context, true)),
-        TextButton(
+        ElevatedButton(
```

Comment:
Can you show me in the Material Design spec where this applies?
https://material.io/components/dialogs#anatomy

Reply:
No please

Reply:
I will change it back to what you originally did

Reply:
I just made a new commit to reset it to the default

Reply:
![Screenshot_20210630-011154](https://user-images.githubusercontent.com/19711677/123910647-60cb4280-d940-11eb-8dd4-1c84b7321035.png)